packages/frontend-core/src/components/grid/cells/HeaderCell.svelte

```diff
@@ -118,6 +120,33 @@
     open = false
   }
 
+  const duplicateColumn = async () => {
+    open = false
+
+    // Generate new name
+    let newName = `${column.name} copy`
+    let attempts = 2
+    while ($definition.schema[newName]) {
+      newName = `${column.name} copy ${attempts++}`
```

Comment:
What happens when it runs out of attempts, does it just throw an error?

Reply:
Attempts increments, so it never runs out!

Reply:
Ah yes, nice!